Daniel Payne Assignment first commit

payneda opened the pull request:
First commit with added methods fudge and loaded and die. not working at
the moment. but free of errors.?????
WIP

payneda committed:
Daniel Payne Assignment first commit

First commit with added methods fudge and loaded and die. not working at
the moment. but free of errors.?????
WIP

ninjapanzer commented on src/dnddiceroller/Die.java:
```diff
@@ -1,12 +1,24 @@
  */
-public class Die {
-  
+public class Die extends NumberedDie{
```
you need to use `interfaces` not `extends`

payneda committed:
New Changes

Added the interface.

payneda committed:
Starting over

payneda committed:
New Changes WIP

Started over again and made the corrected changes. This is still a work
in progress.

payneda committed:
Assignment 8

Final version. Added changes to Fudge and Loaded.
